test/SvgGenerationTest.hx

```diff
@@ -134,9 +140,16 @@ class SvgGenerationTest
         var pngName = generatePng(svgName, pngWidth, pngHeight);
         compareGeneratedToExpected(svgName, pngName);
     }
-    
+
+    private function generateAndCompareTo(svgName1:String, svgName2:String)
+    {
+        var pngName1 = generatePng(svgName1);
+        var pngName2 = generatePng(svgName2, 0, 0, "red");
```

Comment:
Changed to `generateAndCompareWithLayerFilter(svgName1:String, layerId1:String, svgName2:String, layerId2:String)` with layerId as parameters